dags/common/parsing/xml_extractors.py

```diff
@@ -1,10 +1,68 @@
 import xml.etree.ElementTree as ET
 
+from common.constants import WHITE_SPACES
 from common.parsing.extractor import IExtractor
 from common.utils import check_value
 from structlog import get_logger
 
 
+class ContentExtractor(IExtractor):
```

Comment:
where do you use this extractor?

Reply:
I will use it it Abstracts, if we will keep the regex idea